Owncloud iOs Client/AppDelegate.m

```diff
@@ -1721,14 +1710,16 @@ - (void)checkIfIsNecesaryShowPassCode {
             self.window.rootViewController = rootController;
             [self.window makeKeyAndVisible];
             
-            if (IS_IPHONE) {
-                
-                [rootController presentViewController:oc animated:YES completion:nil];
-                
-            } else {
+            if (!IS_IPHONE) {
                 oc.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
                 oc.modalPresentationStyle = UIModalPresentationFormSheet;
-                [rootController presentViewController:oc animated:NO completion:nil];
+                
+            }
+            
+            if (_isPasscodeVisible == false){
```

Comment:
As you wish, I like both, with the equal is more understable but I will change because I OC iOS project there are more ! that == :)